src/components/TransactionRouteList/index.tsx

```diff
@@ -164,23 +176,23 @@ const TransactionRouteList = () => {
 
   const renderFailedView = () => (
     <div>
-      <button type="button" className="btn" onClick={tryAgain}>
+      <button
+        type="button"
+        className="btn"
+        onClick={() => send({ type: "Retry" })}
```

Comment:
Don't use inline functions. Check what will happen when you use inline functions.